db.go

```diff
@@ -184,7 +187,9 @@ func (h *Headscale) initDB() error {
 		return err
 	}
 
-	err = db.AutoMigrate(&Namespace{})
+	_ = db.Migrator().RenameTable("Namespace", "User")
```

Comment:
Should we add a warning message if this migration happens? 

e.g., "Namespaces have been renamed into Users, to better represent Tailscale.com naming".

Reply:
if we migrate, then it is already migrating, a warning is a bit late at that point?

Reply:
I know. It is meant for UX, so they are aware that now Namespaces are Users. 

Not everyone will be reading the changelog I am afraid.